pkg/config/engine/crio/crio.go

```diff
@@ -30,6 +30,17 @@ type Config struct {
 	Logger logger.Interface
 }
 
+type crioRuntime struct {
```

Comment:
Can we add the below to ensure this new type adheres to the engine.RuntimeConfig interface?
```
var _ engine.RuntimeConfig = (*crioRuntime)(nil)
```